tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -4,14 +4,20 @@
  * @group scripts
  */
 class Tests_Dependencies_Scripts extends WP_UnitTestCase {
-	var $old_wp_scripts;
+	public $old_wp_scripts;
+	public static $asset_version;
+
+	public static function setUpBeforeClass() {
+		self::$asset_version = preg_replace( '#\+.*$#', '', classicpress_version() );
```

Comment:
Seems that caching that value can improve the performance or not?
After all we are avoiding to do it always.

Reply:
The only reason I put this statement here was that `self::$asset_version` is much nicer to read and easier to update than `preg_replace( '#\+.*$#', '', classicpress_version() )` in all the other places where we need this value.